terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -40,6 +40,10 @@ route:
       match:
         deployment: prod
         severity: p1
+    - receiver: "verify-p2"
+      match:
+        deployment: integration
+        severity: p2
```

Comment:
this won't work for us. the logic is:

Alert `HubSamlProxyNonEidasErrorRate` needs to go to Pagerduty P1 when `deployment: prod` but to Pagerduty P2 when `deployment: integration`.  Currently, `HubSamlProxyNonEidasErrorRate` is marked `severity: p1` which goes to Pagerduty in prod and Slack in all other environments.

Now it's probably becoming clear that the `severity: p1` label is misnamed.

Anyway the shortest possible change here is:
```suggestion
        severity: p1
```
which will Do The Right Thing, but it's confusing having a `severity: p1` going to a `verify-p2` receiver, and someone will probably "fix" it at some point, so i think there's a future piece of work to find better words to describe what `severity` is trying to do.

Reply:
But whatever alert should go to PagerDuty as a P2 if it’s in office hours and in the integration environment, as I understand it.

Currently integration issues show up in Slack a P1 which confuses people (myself included) when we don’t get called about it.

How do we configure in-hours integration alerts as P2 alerts sending to PagerDuty and Slack, and P1 alerts for production, in this case?

Reply:
@issyl0 that's exactly what my proposed change does:

If it matches `severity:p1` and `deployment:prod`, it goes to pagerduty P1
If it matches `severity:p1` and `deployment:integration`, it goes to pagerduty P2 (which only calls in hours)
If it matches `severity:p1` but has another deployment, it goes to slack.